templates/common/cloud-platform-alt-dns/files/coredns.yaml

```diff
@@ -0,0 +1,113 @@
+mode: 0644
+path: {{ if gt (len (cloudPlatformAPIIntLoadBalancerIPs .)) 0 -}} "/etc/kubernetes/manifests/coredns.yaml" {{ else }} "/etc/kubernetes/disabled-manifests/coredns.yaml" {{ end }}
+contents:
+  inline: |
+    kind: Pod
+    apiVersion: v1
+    metadata:
```

Comment:
This manifest here deserves the `target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'` annotation I mentioned above. You have it in `manifests/` but not in `templates/`, I guess that was a simple oversight

Reply:
Now, both of them have it. Thanks!